packages/cursorless-engine/src/processTargets/modifiers/ItemStage/tokenizeRange.ts

```diff
@@ -41,8 +41,8 @@ export function tokenizeRange(
   const lexemes = text
     // NB: Both the delimiters and the text between them are included because we
     // use a capture group in this split regex
-    .split(/([,(){}<>[\]"'`]|\\"|\\'|\\`)/g)
-    .filter((lexeme) => lexeme.length > 0);
+    .split(/([,(){}<>[\]"'`])|(?<!\\)(\\"|\\'|\\`)/g)
+    .filter(Boolean);
```

Comment:
Why the switch to boolean?

Reply:
Some of the items could be undefined. It's just a more terse way to do `.filter((lexeme) => lexeme != null && lexeme.length > 0);`

Reply:
I generally lean toward verbose, and the conversion to boolean behaviour can be a bit unintuitive i believe. Aren't there strange things where the string "0" evaluates to false or something like that?

Reply:
I don't believe so but I have no problem being more verbose if you prefer. Updated.